src/pe/options.rs

```diff
@@ -3,11 +3,19 @@
 pub struct ParseOptions {
     /// Wether the parser should resolve rvas or not. Default: true
     pub resolve_rva: bool,
+    /// Whether or not to parse attribute certificates.
+    /// Set to false for in-memory representation, as the [loader does not map this info into
+    /// memory](https://learn.microsoft.com/en-us/windows/win32/debug/pe-format#other-contents-of-the-file).
+    /// For on-disk representations, leave as true. Default: true
+    pub parse_attribute_certificates: bool,
```

Comment:
Nit: it would perhaps be slightly better to have the fields in ascending alphanumeric order (that is, `parse_attribute_certificates` before `resolve_rva`).